frontend/factory-ops/src/component/DashBoard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function DashBoard({ onViewChange }) {
```

Comment:
Maybe a different name? A dashboard is a kind of control panel, but we only have buttons to pick content. Perhaps 'ContentPicker' would be more accurate?

Reply:
You have right, i'll change name at the next PR

Reply:
Why not in this? :)

Reply:
I don't think it's that important to change it right now, in my opinion it can safely wait until the next update

Reply:
Naming things is one of the most important aspects in code.

At the very least, add a commented Todo in the code.
`some code // TODO: do some things later `